apps/meteor/client/sidebar/RoomList/SideBarItemTemplateWithData.js

```diff
@@ -136,7 +136,11 @@ const propsAreEqual = (prevProps, nextProps) => {
 	if (prevProps.room._updatedAt?.toISOString() !== nextProps.room._updatedAt?.toISOString()) {
 		return false;
 	}
-	if (prevProps.room.lastMessage?._updatedAt?.toISOString() !== nextProps.room.lastMessage?._updatedAt?.toISOString()) {
+	if (
+		typeof prevProps.lastMessage?._updatedAt === 'object' &&
+		typeof nextProps.lastMessage?._updatedAt === 'object' &&
+		prevProps.lastMessage?._updatedAt?.toISOString() !== nextProps.lastMessage?._updatedAt?.toISOString()
```

Comment:
Since you're checking that lastMessage?._updated is of type object (so it exists), does typescript still requires you to write the ? op?

Reply:
yeah, ideally it shouldn't but since I copied the earlier condition, so those checks still exist there. But now since I've removed this object check, this won't be required